exporter/containerimage/export.go

```diff
@@ -46,11 +46,12 @@ const (
 )
 
 type Opt struct {
-	SessionManager *session.Manager
-	ImageWriter    *ImageWriter
-	Images         images.Store
-	RegistryHosts  docker.RegistryHosts
-	LeaseManager   leases.Manager
+	SessionManager   *session.Manager
+	ImageWriter      *ImageWriter
+	Images           images.Store
+	RegistryHosts    docker.RegistryHosts
+	LeaseManager     leases.Manager
+	ProgressCallback ProgressCallback
```

Comment:
Relying on in-memory callbacks doesn't help with the future path of removing the buildkit vendor from Moby and running buildkitd on its own process.

Reply:
Removed this.